components/search/list-renderer/human-donor.js

```diff
@@ -13,10 +13,21 @@ import {
 
 export default function HumanDonor({ item: humanDonor, accessoryData }) {
   const ethnicities =
-    humanDonor.ethnicity?.length > 0 ? humanDonor.ethnicity.join(", ") : "";
+    humanDonor.ethnicities?.length > 0 ? humanDonor.ethnicities.join(", ") : "";
   const sex = humanDonor.sex || "";
   const title = [ethnicities, sex].filter(Boolean);
   const lab = accessoryData?.[humanDonor.lab];
+  const collections =
+    humanDonor.collections?.length > 0
+      ? `${humanDonor.collections.join(", ")}`
```

Comment:
Technically, you can just use `? humanDonor.collections.join(", ")`. The string-template syntax really only helps for composing strings from multiple elements, but after the `.join()` you only have one element. It doesn’t make any execution difference, but would have saved a little typing.